### PR TITLE
Adds React Helmet

### PR DESCRIPTION
**Pull request by DLMR**
With this change the bundle size will be increased by about 25 kB. We should also look into if we should add base and scripts to the configuration to be used with React Helmet.

* Removed DocumentTitle and changed how the template is used together with the application wrapper. 
* Added new configuration options for meta and links.
* Removed default styles.

#### Todo
* [x] Handle default styles in `roc-web-react`.
* [x] Discuss the API as mentioned above.

### CODE REVIEW

**Comment by DLMR** [comment]
I think we will look at the documentation in another PR.

File: src/roc/config/roc.config.js [hunk 1]
```diff
@@ -1,6 +1,16 @@
 const config = {
     stats: 'build/client/webpack-stats.json',
     applicationName: '',
+    meta: [{
```
Comment (andreasrs) [inline src/roc/config/roc.config.js]: It seems this is currently not injected into the application. Thoughts/intentional?

Comment (DLMR) [inline src/roc/config/roc.config.js]: I had a bug around this but thought it was solved... Will take another look!

Comment (andreasrs) [inline src/roc/config/roc.config.js]: It was local setup here. Good that you bring the Helmet-issue to their team.

**Pull request merged**